brigade/static/scss/atoms/_button.scss

```diff
@@ -0,0 +1,9 @@
+@import '../core/variables';
+
+//
+// Button
+//
+
+.button {
+  box-sizing: border-box !important; // TODO: Get rid of !important (the old CfA stylesheet we're using right now overides the button box-sizing)
```

Comment:
Most html5 reset stylesheets apply this to all elements... we should consider doing the same (subsequently).